src/mpp/Dec.hpp

```diff
@@ -857,17 +915,16 @@ constexpr enum path_item_type get_next_arr_item_type()
 template <class ARR, enum path_item_type TYPE>
 constexpr size_t get_next_arr_static_size()
 {
-	if constexpr (TYPE <= PIT_STADYN)
+	if constexpr (TYPE > PIT_BAD && TYPE <= PIT_STADYN)
```

Comment:
Nit: looks like an unrelated change.

Reply:
Build fails with compilation error without this change.